src/pkg/clients/Redis.go

```diff
@@ -15,15 +16,21 @@ type IRedisClient interface {
 var _ IRedisClient = redisClient{}
 
 type redisClient struct {
-	redisClient *redis.Client
+	redisClient *redis.ClusterClient
 }
 
-func NewRedisClient(addr, pwd string, db int) IRedisClient {
+func NewRedisClient(addr, pwd string, configDev bool) IRedisClient {
```

Comment:
Instead of ConfigDev, I think it's better if we use something like `UseTLS`. It represents more how it behaves.